images/debian-python/Dockerfile.debian

```diff
@@ -39,6 +39,8 @@ RUN git clone https://github.com/pyenv/pyenv .pyenv \
   && pyenv install ${PYTHON_VERSIONS} \
   && pyenv global $(echo ${PYTHON_VERSIONS} | awk '{ print $NF }')
 
+RUN pip install poetry
```

Comment:
Saves having to do this in every CodePipeline Action.

Reply:
Not sure we get the benefit of this in the regression tests now we are using a virtual environment. It's not doing any harm, but we could revert the change if anyone thinks that's worth it?

Reply:
I'd take it out if it's not actually used.